doc/release/2.11.0.rst

```diff
@@ -0,0 +1,586 @@
+
+1.  To create a read view, call the ``box.read_view.open()`` function:
+
+        .. code-block:: tarantoolsession
+
+            tarantool> read_view1 = box.read_view.open({name = 'read_view1'})
+
+2.  After creating a read view, you can access database spaces and their indexes and get data using the familiar ``select`` and ``pairs`` data-retrieval operations, for example:
+
+        .. code-block:: tarantoolsession
+
+            tarantool> read_view1.space.bands:select({}, {limit = 4})
+            ---
+            - - [1, 'Roxette', 1986]
+              - [2, 'Scorpions', 1965]
+              - [3, 'Ace of Base', 1987]
+              - [4, 'The Beatles', 1960]
+
+3.  When a read view is no longer needed, close it using the ``read_view_object:close()`` method:
+
+        .. code-block:: tarantoolsession
+
+            tarantool> read_view1:close()
+
+To learn more, see the `Read views <https://www.tarantool.io/en/enterprise_doc/read_views/>`_ topic.
+
+
+.. _data_compression_improvements:
+
+Data compression improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tarantool Enterprise now includes the ``zlib`` algorithm for `tuple compression <https://www.tarantool.io/en/enterprise_doc/tuple_compression/>`_.
+This algorithm shows good performance in data decompression,
+which reduces CPU usage if the volume of read operations significantly exceeds the volume of write operations.
+
+To use the new algorithm, set the ``compression`` option to ``zlib`` when formatting a space:
+
+.. code-block:: lua
+
+    box.space.my_space:format{
+        {name = 'id', type = 'unsigned'},
+        {name = 'data', type = 'string', compression = 'zlib'},
+    }
+
+The new ``compress`` module provides an API for compressing and decompressing arbitrary data strings using the same algorithms available for tuple compression:
+
+.. code-block:: lua
+
+    compressor = require('compress.zlib').new()
+
+    data = compressor:compress('Hello world!') -- returns a binary string
+    compressor:decompress(data) -- returns 'Hello world!'
+
+
+
+.. _wal_extensions:
+
+WAL extensions
+~~~~~~~~~~~~~~
+
+Tarantool can use a write-ahead log not only to maintain data persistence and replication.
+Another use case is implementing a CDC (Change Data Capture) utility that transforms a data replication stream and provides the ability to replicate data from Tarantool to an external storage.
+
+.. image:: _images/wal_extensions.svg
+    :align: center
+
+With v2.11, Tarantool Enterprise provides WAL extensions that add auxiliary information to each write-ahead log record.
+For example, you can enable storing old and new tuples for each write-ahead log record.
+This is especially useful for the ``update`` operation because a write-ahead log record contains only a key value.
+
+See the `WAL extensions <https://www.tarantool.io/en/enterprise_doc/wal_extensions/>`_ topic to learn how to enable and configure WAL extensions.
+
+
+
+.. _community:
+
+Community
```

Comment:
maybe use the full product name?
Just "Community" as a heading is ambiguous. Is it about people who use Tarantool?

Reply:
Agree, fixed both Enterprise and Community titles